src/const.ts

```diff
@@ -11,17 +12,18 @@ export const GITHUB_ISSUES_URL = "https://github.com/ddev/ddev/issues"
 export const DOCUMENTATION_URL = "https://ddev.readthedocs.io/"
 export const DISCORD_URL = "/s/discord"
 export const EMAIL_URL = "mailto:support%40ddev.com"
```

Comment:
This typo to be specific we need to replace %40 for @

Reply:
This works in regular links, and the idea is to prevent web crawlers from capturing the email address.

Reply:
oooh, that makes sense, I thought this was the original issue with the email on the footer.